themes/default/content/blog/managing-github-with-pulumi/index.md

```diff
@@ -0,0 +1,599 @@
+---
+title: "How we manage GitHub at Pulumi with Pulumi"
+date: 2022-03-18T15:29:33-07:00
+meta_desc: In this article, Guinevere Saenger describes how Pulumi migrated management of GitHub team permissions using Pulumi all the way down.
```

Comment:
ah, my bad! I thought it wasn't strictly necessary!

I'm a bit confused where it would come into play, too - the social cards in the preview all have the Pulumi logo on them...

Reply:
@guineveresaenger they appear on the blog list page and also on social
https://www.pulumi.com/blog/